.gitlab-ci.yml

```diff
@@ -682,6 +682,35 @@ zombienet-tests-parachains-disputes:
   tags:
     - zombienet-polkadot-integration-test
 
+zombienet-test-parachains-upgrade-smoke-test:
+  stage:                           deploy
```

Comment:
```suggestion
  stage:                           stage4
```
Stages were renamed and became artificial in order to use DAG in the pipeline.

Reply:
why stage4? Shouldn't it be the same stage as other zombienet tests, which is stage3?

Reply:
fully agree, fixed suggestion